lib/parity/environment.rb

```diff
@@ -57,9 +57,9 @@ def deploy_to_heroku
     end
 
     def restore
-      if production?
+      if production? && !forced?
```

Comment:
I really like the evolution of this. Can we make this method `not_forced?` and move the `!` into the checking method so that the conditions being evaluated here are as easy to read as possible?